src/maxminddb.c

```diff
@@ -934,10 +937,7 @@ LOCAL int find_address_in_search_tree(MMDB_s *mmdb, uint8_t *address,
     uint16_t start_bit = max_depth0;
 
     if (mmdb->metadata.ip_version == 6 && address_family == AF_INET) {
-        int mmdb_error = find_ipv4_start_node(mmdb);
```

Comment:
find_ipv4_start_node already gets called from MMDB_open if ip_version == 6. It doesn't appear that this method would need to be called again.